src/compass/utils/geo_runconfig.py

```diff
@@ -64,6 +65,14 @@ def load_from_yaml(cls, yaml_path: str, workflow_name: str) -> GeoRunConfig:
         geocoding_dict = groups_cfg['processing']['geocoding']
         check_geocode_dict(geocoding_dict)
 
+        # Check troposphere weather model file if not None. This
+        # troposphere correction is applied only if this file is not None
+        weather_model_path = groups_cfg['dynamic_ancillary_file_group'][
+            'weather_model_file'
+        ]
+        if weather_model_path is not None:
+            check_file_path(weather_model_path)
```

Comment:
In case the weather model file is not provided, we can still apply troposphere delay using static model in python level because we have all ingredients for the calculation. What do you think?

Reply:
I agree. I think we should apply the static troposphere correction regardless. Only, have we decided if we want to proceed with the computation of the static delay at the core module or at a Python level? I am in favor of the second, it should not be that expensive to compute anyway